static/js/compose_actions.js

```diff
@@ -155,7 +162,17 @@ export function update_narrow_to_recipient_visibility() {
             $(".narrow_to_compose_recipients").show();
             return;
         }
+    } else if (message_type === "private") {
+        const $to_text = $(".to_text");
+
+        if (composing_to_current_pm_narrow()) {
+            $to_text.addClass("disable_events");
+            return;
+        }
+        $to_text.removeClass("disable_events");
```

Comment:
Using a class here is reasonable; but I don't think just disabling pointing events is correct; there shouldn't be special hover behavior if it's not a button.

It might be cleaner to invert the meaning -- adding/removing the `narrow_to_compose_recipients` class on it, for example, and having the hover/pointer/click handler behavior all be dependant on that being present.

Reply:
In a previous comment I mentioned that I am now using the class you suggested. This let me overload the existingc click handlers and tippy behavior. Much easier! Thank you!!